### PR TITLE
Add git autoupdate config in rainbow

### PR DESCRIPTION
**Pull request by LeaYeh**

### CODE REVIEW

File: ajax/libs/rainbow/package.json [hunk 1]
```diff
@@ -20,5 +20,13 @@
...
       "url": "https://github.com/ccampbell/rainbow.git"
     }
-  ]
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ccampbell/rainbow.git",
+    "basePath": "",
+    "files": [
+      "*.js"
```
Comment (PeterDaveHello) [inline ajax/libs/rainbow/package.json]: You sure about this?

**Comment by LeaYeh** [comment]
check it.

File: ajax/libs/rainbow/package.json [hunk 2]
```diff
@@ -20,5 +20,15 @@
...
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/ccampbell/rainbow.git",
+    "basePath": "",
+    "files": [
+      "themes/*",
+      "js/*.js",
+      "js/language/*"
+    ]
```
Comment (PeterDaveHello) [inline ajax/libs/rainbow/package.json]: Let's use `js/**/*.js` instead of `js/*.js` + `js/language/*`.

**Comment by LeaYeh** [comment]
check it

**Pull request merged**